### PR TITLE
Issue #: 30, 31 & 32

### PR DESCRIPTION
**Pull request by aelsammak**
Enabled CORS Globally, resolved issue with the JPA mapping of Entities and added new endpoints to the SurveyorController
This is the database currently for reference, I did not change how the database looks...but, I did change the way the models were being mapped because they were giving me errors. The only thing I noticed from #22 was why did we remove the LongAnswerQuestion class?

current_DB

### CODE REVIEW

File: src/main/java/ca/group20/sysc4806project/model/answer/Answer.java [hunk 1]
```diff
@@ -58,11 +55,11 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) return false;
         Answer that = (Answer) o;
```
Comment (Erik-Iuhas) [inline src/main/java/ca/group20/sysc4806project/model/answer/Answer.java:56]: Personally, I would change "that" variable name as it is a bit unclear.

**Review by Erik-Iuhas — COMMENTED** [review]
Looks good to me for the most part.

**Review by mariannebrissette3 — APPROVED** [review]
Looks good!

**Comment by aelsammak** [comment]
I believe that for now, we can leave the get functions for the ids until we have a need for having getters for the objects.

**Review by Daniel-W-Innes — APPROVED** [review]

**Pull request merged**